### PR TITLE
Allow NUM_CYCLES to be optionally set by command line argument.

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
NUM_CYCLES is defined in the *Mon/config or user settings file and sets the number of cycles used in time series plots.  It's sometimes useful to plot specific sources at a different period for time series, while using the default for all others.  This change provides a means to easily accomplish that.

Testing was done on Orion, Hera, and wcoss2 confirming both the NUM_CYCLES values with and without command line setting.  Time series plots were made for RadMon and OznMon confirming the correct use of NUM_CYCLES.

Note that the MinMon does not create time series plots.  Note too that the ConMon is still waiting update to use the shared GSI-monitor resources (#28) and does not plot correctly at the moment.  Confirmation of the correct setting of NUM_CYCLES in the ConMon plots was made though.

This completes #38.

### CODE REVIEW

File: src/Ozone_Monitor/image_gen/ush/OznMon_Plt.sh [hunk 1]
```diff
@@ -93,7 +72,7 @@ do
    shift
 done
 
-if [[ $nargs -lt 0 || $nargs -gt 9 ]]; then
+if [[ $nargs -lt 0 || $nargs -gt 13 ]]; then
```
Comment (DavidHuber-NOAA) [inline src/Ozone_Monitor/image_gen/ush/OznMon_Plt.sh:75]: Should this be 11?

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good.  Just one trivial comment.

File: src/Ozone_Monitor/image_gen/ush/OznMon_Plt.sh (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply src/Ozone_Monitor/image_gen/ush/OznMon_Plt.sh:75]: No, 13 is right.  The previous value of 9 was wrong and should have been 11.  The options include run, pdate, ncyc, and tank, plus are 2 older, unique options for comparison plotting (comp1, comp2).  So total of 12 + suffix is possible.

Comment (DavidHuber-NOAA) [reply src/Ozone_Monitor/image_gen/ush/OznMon_Plt.sh:75]: Okie dokie.  Thanks!

**Pull request merged**